No reason to install an autoconf-generated config.h file which isn't referenced
by any of the other headers we install, or by applications, and possible
conflicts between definitions of variables on biarch systems is a reason not to.

--- Linux-PAM-0.77/Makefile	2003-10-23 08:51:11.000000000 -0400
+++ Linux-PAM-0.77/Makefile	2003-10-23 08:50:59.000000000 -0400
@@ -55,7 +55,7 @@
 
 install: _pam_aconf.h prep
 	$(MKDIR) $(FAKEROOT)$(INCLUDED)
-	$(INSTALL) -m 444 security/_pam_aconf.h $(FAKEROOT)$(INCLUDED)
+	: $(INSTALL) -m 444 security/_pam_aconf.h $(FAKEROOT)$(INCLUDED)
 	for x in $(THINGSTOMAKE) ; do $(MAKE) -C $$x install ; done
 
 remove:
